### ISSUE TITLE
Missing Spawns

### ISSUE THREAD

**Issue by Kasaichan** [MineInAbyss/Issue-Tracker#140]
### Expected behavior

I expected to encounter more ready mobs in-game.

### Observed/Actual behavior

I have looked through the github config files for spawn conditions, compared it to notion & the resource-pack, and found several missing mobs: Hamashirama, Hisomukuro, Spiralsquid, Billabonk. Files for the Amakagame were found as well, but will be an honorable mention due to the set spawn conditions it shares with the Guild Master, who also disappeared in-game.

### Steps/models to reproduce

No new mob models have been added since February 10th 2022, and no new mobs have been added since February 21st 2021. Most recent mob reinsertion was on April 4th 2022. [A link to a summarized github spawn table as of May 14th 2022 is viewable here.](https://docs.google.com/document/d/1EBIrS2zTWrGsLC6S3RirRWm7sxe4F8cJ9OkFqaB6suE/edit?usp=sharing)

### Other

Given directly to the importer was [this file](https://github.com/MineInAbyss/Issue-Tracker/files/8813732/MobTrioVol.1.zip) containing the assets for the spiralsquid and other mobs left unused to this day (those being the old hermit rat and dokisaru). Even though the swim animation was provided, it was not used when it was converted into a single-bone modelengine .json, leaving the entity discarded since it was given on December 14th, 2021.

### Agreements

- [X] I have searched for and ensured there isn't already an open issue regarding this.
- [X] I have made a detailed description of the bug so that it can be reproduced.

**Comment by Boy0000** [comment]
Hisomukuro's AI and pathfinding is still not done
Spiralsquid is gonna come with the others here
Billabonk is waiting on animations but mainly 1.19 frogs and then to mimic their AI and stuff
Amakagame just wont be used as for a while as it is meant to spawn in predefined areas which we currently havent got supported. And no its not the same as Guild Master as he is manually spawned.

Bruddah guild master is and has been in Guild HQ since he was put in in April or whenever we added Guilds.

When the AI isnt half working, bugs with the vanilla replacements are fixed (like hostile mobs breaking doors etc) and well 1.19 for billabonk we might finish them up. But right now its not a priority to finish these mobs up. Hisomukuro is the only real unique mob but again, fishAI is clunky etc
